client/apps/atomic_search_widget/app.js

```diff
@@ -172,18 +172,30 @@ function ajEnableListener() {
 }
 
 function getToolUrl() {
-  const baseSelector = 'a:contains("Search")';
-  // this could be within a course or subaccount
-  const localNavElement = $(`#section-tabs ${baseSelector}`);
-  const globalNavElement = $(`#menu ${baseSelector}`);
+  const searchWords = ['Search', 'Søk', 'Buscar', 'Rechercher', 'Търсене', 'Cerca', 'Hledat', 'Søg', 'Suche', 'Αναζήτηση', 'Ikastaro', 'Etsi', 'Cuardaigh', 'सभी', 'Keresés', 'Որոնել', 'Cari', 'Cerca', 'Hemî', 'Doorzoek', 'Pesquisar', 'Căutați', 'Поиск', 'Sök', 'Tüm'];
+  let url;
+  // the linter doesn't like normal for loops, we use 'some' because returning
+  // true breaks you out early
+  searchWords.some((word) => {
+    const baseSelector = `a:contains("${word}")`;
+    // this could be within a course or subaccount
+    const localNavElement = $(`#section-tabs ${baseSelector}`);
+    const globalNavElement = $(`#menu ${baseSelector}`);
 
-  if (localNavElement.attr('href')) {
-    return localNavElement.attr('href');
-  } else if (atomicSearchConfig.accountId && atomicSearchConfig.externalToolId) {
-    return `/accounts/${atomicSearchConfig.accountId}/external_tools/${atomicSearchConfig.externalToolId}`;
-  }
+    if (localNavElement.attr('href')) {
+      url = localNavElement.attr('href');
+      return true;
+    } else if (atomicSearchConfig.accountId && atomicSearchConfig.externalToolId) {
```

Comment:
I think this check should happen after the loop because if it is in a non-english language, it won't the localNav element the first time the list and then default to the global search which it shouldn't